medusa/helpers.py

```diff
@@ -716,7 +716,7 @@ def sanitize_scene_name(name, anime=False):
         name = name.replace(x, '')
 
     # tidy up stuff that doesn't belong in scene names
-    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.')
+    name = name.replace('- ', '.').replace(' ', '.').replace('&', 'and').replace('/', '.').replace(': ', '-')
```

Comment:
Is there any benefit of replacing with `-` over just a space?  I would think with a space it would return results for any of:
- `Mythbusters: The Search` 
- `Mythbusters-The Search`
- `Mythbusters, The Search`
- `Mythbusters The Search`